pkgs/top-level/lua-packages.nix

```diff
@@ -213,15 +241,24 @@ let
         LUA_LIBDIR="-L${lua}/lib"
         );
     '';
+
+    meta = with stdenv.lib; {
+      description = "PTY control for Lua";
+      homepage = "http://www.tset.de/lpty";
+      license = licenses.mit;
+      maintainers = with maintainers; [ vyp ];
+      platforms = platforms.linux;
+    };
   };
 
   lua-iconv = buildLuaPackage rec {
     name = "lua-iconv-${version}";
     version = "7";
+
     src = fetchFromGitHub {
       owner = "ittner";
       repo = "lua-iconv";
-      rev = "e8d34024a6b185a759733915f116cc5588550261";
+      rev = name;
```

Comment:
@vyp Curious why this is changing?

Reply:
(Since the tag could be moved, which would then break the sha256)

Reply:
Since the commit hash was tagged with that, it's simpler to just use the tag name instead. This is the usual case with most other nix package expressions, since the upstream authors usually follow the same pattern for naming new releases and tags, updating a package is usually a simple change to `version` and the `src.sha256`. Usually tags never move so it's fine. Does that explain?

If you want I could revert this but it would be irregular. In fact I'm surprised I didn't pick this up when reviewing https://github.com/NixOS/nixpkgs/pull/29949.

Reply:
To be honest, my last commit here should have really been part of a separate PR I think, but oh well. :/

Reply:
No need to revert, I'm the newcomer here :smile:
I was just curious, thanks for explaining.